### ISSUE TITLE
[$10][MSFT-228] "Topcoder handle" name foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2886]
**Steps to Reproduce**
1. Go to: https://www.topcoder.com/members/doremihong
2. Right click > Inspect Elements 
3. Check the "Topcoder handle" name

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change text color to #707594. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: rgb(157, 159, 160) = #9D9FA0
Background Color: #FFFFFF
Contrast Ratio: 2.65:1
=> Therefor, "Topcoder handle" name foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: 

**WCAG Category**: 1. Perceivable
**WCAG Level**: AAA
**WCAG Success Criterion**: 1.4.6 Contrast (Enhanced)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F29

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096244 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096244 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096244 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/2950

**Comment by PrakashDurlabhji** [comment]
@gurmeetb PR updated

**Comment by codeMinter** [comment]
@PrakashDurlabhji will check in sometime.

**Comment by Dara-K** [comment]
Please use #555555 color for the handle.

**Comment by tosha5252** [comment]
Unable to verify at topcoder-dev.com...receiving a 404 error when attempting to access member profiles.

**Comment by tosha5252** [comment]
color set to rgb(157, 159, 160) = #9D9FA0, please change to #555555, per DaraK's comment above

**Comment by tosha5252** [comment]
Env:

Windows 10
Browser: Chrome

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096759 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096759 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
Env:
Windows 10
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096759<br/><br/>```This is an automated message for crazyk via Topcoder X```
